jax/_src/lax/lax.py

```diff
@@ -591,10 +591,10 @@ def conv_general_dilated(
     rhs_dilation = (1,) * (rhs.ndim - 2)
   if isinstance(padding, str):
     lhs_perm, rhs_perm, _ = dnums
-    rhs_shape = np.take(rhs.shape, rhs_perm)[2:]
+    rhs_shape = np.take(rhs.shape, rhs_perm)[2:]  # type: ignore[index]
```

Comment:
I'm concerned that most of these look like either NumPy bugs or mypy bugs, in the sense that valid code is being rejected by inaccurate type annotations or inadequate type inference. File upstream bugs? The NumPy folks say in their release notes they want feedback on type annotations.

Reply:
you are right, but there was just too much going on with this numpy upgrade. Now that this has calmed down I will try to file a bug upstream. I am not sure yet if it is a mypy only problem or a pytype also.